data/hai/hai reveal 4 middle.txt

```diff
@@ -966,8 +966,30 @@ mission "Hai Reveal [A08] Symbol Identification"
+			`	Moscow is thousands of kilometers away, so you travel most of the route on Shuttles. It seems some new pilots took an option you never considered: planet-side travel in an exclusive contract with local transit authorities who pay half the mortgage. Certainly a safe career, but not as exciting as your experiences.`
+			`	You fly past mountain ranges, lakes, and deserts, and the occasional tundra. Everywhere, you see the gray cities of Earth packed in their ancient locations. Even from kilometers above, the cities' vast megabuildings are an imposing sight. There's still some farmland, even now that farming has largely moved off-world; but as you well know, all of that isn't enough for the twenty billion people living on Earth.`
+			`	After three shuttle trips, you land in the city of Nizhny, and board a two-story train that hovers in a tunnel of superconducting magnets. Such a complex system would be unprofitable anywhere else in human space, but the population is so high here that these village-sized trains run non-stop at full capacity.`
+			choice
+				`	(Exit the train at Moscow.)`
+			`	Strolling through Moscow to your destination, you pass by many historic sites. Saint Basil’s Cathedral catches your eye, an ancient religious site that Ivan the Terrible ordered to be built. That was almost one and a half millennia ago, but it has been painstakingly preserved all this time. It is shadowed on three sides by monolithic buildings larger than any ship you've seen, but there is a large array of flood lights keeping the cathedral lit.`
+			`	You notice the building behind the cathedral is your destination. After a brief walk, you pass through its main doors into a room as big as football field, filled with in-building trains, moving walkways, and hover platform elevators. You find an information desk, and inquire the meaning of 'room number: 3017A19' - the 301st floor, section 7A, room 19 - and get directions. It's amazing how large Earth's megabuildings have become. Some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`
+			choice
+				`	(Head to the 301st floor, section 7A, room 19.)`
 			`	Reaching the room in question you knock on the door which has a nameplate that reads, 'Giti Gilani' with the title 'Research Librarian' in small print underneath. You are swiftly greeted by a short, energetic looking lady. She looks you up and down and nods. "Captain <last>, you're not as tall as I thought you'd be in person. Please, please come in."`
+				goto "converse with giti"
+			label "take the direct route"
+			`	You're ushered to a luxury hover-taxi with built-in bar and entertainment. It moves at hypersonic speeds, reaching your destination in under twenty minutes. The taxi attaches its aft exit to the side of the building, about half the way up. The well-dressed driver leads you into the building and escorts to your destination.`
```

Comment:
```suggestion
			`	You're ushered into a luxury hover-taxi, complete with built-in bar and the latest forms of entertainment. It moves at hypersonic speeds, reaching your destination in under twenty minutes. The taxi attaches its aft exit to the side of the building, about half the way up. The well-dressed driver leads you into the building and escorts to your destination.`
```
Is the taxi going through space? Hypersonic speeds over Earth can be dangerous for the people on the ground (glass and popped eardrums). I also don't know how 31st century entertainment looks like but I'll just make it vague and let the player decide what it means.

Reply:
There are experiments today into making supersonic jets that produce almost no boom. It's probably best to remove those details from the paragraph though, plus this is the dialog for players that want to skip the wordy text. I'll update this shortly.

Reply:
It is no longer a hover taxi. Instead, it enters a ballistic orbit into space. I made your other changes (into, complete, the latest forms of) manually.